style.css

```diff
@@ -0,0 +1,187 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins');
+
```

Comment:
The way you import the font is correct but you can also choose the different font styles and this way apply the weights you want to the font.

You select your styles by clicking on "Select this style":
<img width="1350" alt="Untitled" src="https://user-images.githubusercontent.com/66212802/95874286-061d9680-0d71-11eb-8fba-4a66a85f05f2.png">

Once you have the ones you like, you can embed them:
<img width="1350" alt="Untitled copy" src="https://user-images.githubusercontent.com/66212802/95874435-2ea59080-0d71-11eb-8afc-a7ea128f5b69.png">

This is another way of embedding the font (although you can do this as you did but with the different styles selected).
<img width="1350" alt="Untitled copy 2" src="https://user-images.githubusercontent.com/66212802/95874456-3402db00-0d71-11eb-87de-c051e0b7fdd4.png">

Done! :)

Reply:
Thanks Marta! I'm doing again the challenge with all the suggestions you gave me :)